sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/jsonExpressions.scala

```diff
@@ -801,7 +801,7 @@ case class SchemaOfJson(
       }
     }
 
-    UTF8String.fromString(dt.catalogString)
+    UTF8String.fromString(dt.sql)
```

Comment:
Can we update migration guide?

Reply:
Will it break user queries?

Reply:
Nope but the output from this function becomes different

Reply:
Should I add a SQL config, just in case?

Reply:
Okay .. NVM ..